dsas-wps/src/main/java/gov/usgs/cida/dsas/wps/geom/Intersection.java

```diff
@@ -211,7 +211,7 @@ public static SimpleFeatureType buildSimpleFeatureType(SimpleFeatureCollection c
 		SimpleFeatureTypeBuilder builder = new SimpleFeatureTypeBuilder();
 
 		builder.setName("Intersections");
-		builder.add("the_geom", Point.class, crs);
+		builder.add(DEFAULT_GEOM_ATTR, Point.class, crs);
```

Comment:
Thanks for this, I was going to do that but ended up just leaving it as a magic number.